src/jdk.net/windows/classes/jdk/net/WindowsSocketOptions.java

```diff
@@ -51,8 +56,45 @@ boolean getIpDontFragment(int fd, boolean isIPv6) throws SocketException {
         return getIpDontFragment0(fd, isIPv6);
     }
 
+    @Override
+    void setTcpkeepAliveProbes(int fd, final int value) throws SocketException {
+        setTcpkeepAliveProbes0(fd, value);
+    }
+
+    @Override
+    int getTcpkeepAliveProbes(int fd) throws SocketException {
+        return getTcpkeepAliveProbes0(fd);
+    }
+
+    @Override
+    void setTcpKeepAliveTime(int fd, final int value) throws SocketException {
+        setTcpKeepAliveTime0(fd, value);
+    }
+
+    @Override
+    int getTcpKeepAliveTime(int fd) throws SocketException {
+        return getTcpKeepAliveTime0(fd);
+    }
+
+    @Override
+    void setTcpKeepAliveIntvl(int fd, final int value) throws SocketException {
+        setTcpKeepAliveIntvl0(fd, value);
+    }
+
+    @Override
+    int getTcpKeepAliveIntvl(int fd) throws SocketException {
+        return getTcpKeepAliveIntvl0(fd);
+    }
+
+    private static native boolean keepAliveOptionsSupported0();
     private static native void setIpDontFragment0(int fd, boolean value, boolean isIPv6) throws SocketException;
     private static native boolean getIpDontFragment0(int fd, boolean isIPv6) throws SocketException;
+    private static native void setTcpkeepAliveProbes0(int fd, int value) throws SocketException;
+    private static native int getTcpkeepAliveProbes0(int fd) throws SocketException;
```

Comment:
there is inconsistency in function naming some place you use 'keep' and other places you use 'Keep', can you please make it consistence ?.

Reply:
Fixed inconsistency.